.openshift-ci/drivers/drivers.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM pipeline:fc36-base
+
+COPY --from=pre-build:latest /kernel-modules /kernel-modules
+COPY --from=fc36-drivers:latest /kernel-modules /kernel-modules
+COPY --from=fc36-drivers:latest /FAILURES /FAILURES
+COPY --from=rhel8-drivers:latest /kernel-modules /kernel-modules
+COPY --from=rhel8-drivers:latest /FAILURES /FAILURES
+COPY --from=rhel7-drivers:latest /kernel-modules /kernel-modules
+COPY --from=rhel7-drivers:latest /FAILURES /FAILURES
+
+COPY /.circleci/kernel-module-build-failures-check/20-test-for-build-failures.sh /scripts/drivers-build-failures.sh
```

Comment:
Maybe move the scripts you are using from .circlecli to .openshift-ci.

Reply:
If I do that, I will need to also modify `.circleci/config.yaml` or duplicate the script, I think it would be cleaner to do so on a follow up PR once we completely remove CircleCI, WDYT?

Reply:
Makes sense.